package.json

```diff
@@ -45,6 +45,7 @@
     "bluebird": "^3.5.0",
     "content-type": "^1.0.3",
     "portastic": "^1.0.1",
+    "puppeteer": "^0.12.0",
```

Comment:
Puppeteer cannot be in dependencies, otherwise Chromimum would be included even in node-basic docker image. Let's put it to optionalDependencies similarly as selenium-webdriver.